src/components/NavBar/NavBar.tsx

```diff
@@ -4,7 +4,7 @@ export const NavBar = () => {
       <div className="flex-1">
         <a className="btn btn-ghost text-xl">Mayhem Hub</a>
       </div>
-      <div className="flex-none">
+      <div className="hidden flex-none sm:block">
```

Comment:
Is this hidden still meant to be here?

Reply:
Yep, this hides the links on screens that are too small, like phones, this is only until I get a proper dropdown menu working that only shows on mobile screens

Reply:
We have a warning message when its not supported and a link that links to a list of supported browsers already https://github.com/portapack-mayhem/MayhemHub/blob/main/src/components/SerialLoader/SerialLoader.tsx#L64

Which links to here https://caniuse.com/web-serial